src/elements/_has-arrow.scss

```diff
@@ -99,11 +99,12 @@ $border-width: 1px;
     }
 
     @include __arrowBody {
-      background-image: linear-gradient(
-        to bottom left,
-        transparent 50%,
-        var(--hxBackgroundColor, $gray-0) 50%
-      );
+      background-image:
+        linear-gradient(
+          to bottom left,
+          transparent 50%,
+          var(--hxBackgroundColor, $gray-0) 50%
+        );
```

Comment:
Because the function arguments are on separate lines, [`"declaration-colon-newline-after": "always-multi-line"`](https://stylelint.io/user-guide/rules/declaration-colon-newline-after) takes affect, so we have to begin `linear-gradient` on the line after `background-image`.